C/c4ExpiryEnumerator.h

```diff
@@ -36,7 +36,7 @@ extern "C" {
         @param e The enumerator.
         @return A slice representing the doc ID (internal pointer, so don't free)
```

Comment:
This comment needs to be updated, since the caller _does_ need to free the result now.